modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -205,9 +209,13 @@ export class {{classname}} {
             body: formParams.toString(),
 {{/hasFormParams}}
             search: queryParameters,
-            withCredentials:true
+            withCredentials:isWithCredentials
```

Comment:
If it's not defined, then you will set withCredentials to undefined, which exactly replicates what the old code did by not setting it at all. :-)